ext/ftp/tests/ftp_connect_001.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+ftp_connect - return FALSE if connection fails and Waning is generated
+--EXTENSIONS--
+ftp
+--INI--
+max_execution_time=2
```

Comment:
This may cause issues, since the default timeout for `ftp_connect` is 90 seconds. It is probably better not to set the `max_execution_time` at all, but rather to pass a suitable `$timeout` to `ftp_connect()`.

Reply:
I Will adjust